cves/kernel/CVE-2014-3183.yml

```diff
@@ -219,8 +260,9 @@ subsystem:
     e.g.
         name: ["subsystemA", "subsystemB"] # ok
         name: subsystemA # also ok
-  name:
+  name: hid
   note:
```

Comment:
It would be nice for there to be a description of the subsystem in the linux kernel

Reply:
I added some more info.